server/datastore/mysql/policies_test.go

```diff
@@ -35,7 +35,7 @@ func TestPolicies(t *testing.T) {
 		{"MembershipViewNotDeferred", func(t *testing.T, ds *Datastore) { testPoliciesMembershipView(false, t, ds) }},
 		{"TeamPolicyLegacy", testTeamPolicyLegacy},
 		{"TeamPolicyProprietary", testTeamPolicyProprietary},
-		{"ListMergedTeamPolicies", testListMergedTeamPolicies},
+		// {"ListMergedTeamPolicies", testListMergedTeamPolicies},
```

Comment:
Remove?

Reply:
oops, that was a shortcut for testing.  That's reverted.